include/grl/ros/KukaLBRiiwaROSPlugin.hpp

```diff
@@ -330,13 +330,15 @@ namespace grl {
              break;
            case grl::flatbuffer::ArmState_StartArm:
              if (debug) {
-              ROS_INFO("Sending start!");
+               ROS_INFO("Sending StartArm!");
              }
              break;
            case grl::flatbuffer::ArmState_ShutdownArm:
              break;
            default:
-             ROS_INFO("KukaLBRiiwaROSPlugin in unsupported mode!");
+             if(debug) {
```

Comment:
modulo counter sounds better. Also, looks like you have the emergency stop #123 included in this pull request. Not sure if you want a separte merge